fhirclient/client.py

```diff
@@ -28,7 +28,7 @@ class FHIRClient(object):
         - `launch_token`: The launch token
     """
     
-    def __init__(self, settings=None, state=None, save_func=lambda x:x):
+    def __init__(self, settings=None, state=None, save_func=None, load_func=None):
```

Comment:
load_func is misleading, since it does both save and load.  Suggest "cache_func"

Reply:
I think I misread the code.  Thanks for clearing that up.